firefox-ios/nimbus-features/messaging/messaging-evergreen-messages.fml.yaml

```diff
@@ -53,3 +53,21 @@ import:
                 action: OPEN_URL
                 action-params:
                   url: https://www.macrumors.com
+
+              # Serves as an example of how microsurveys may look like
+              microsurvey-message:
```

Comment:
Is the plan to keep the example in the code? We will also need to update the [experimenter](https://experimenter.info/messaging/mobile-messaging) docs (added ticket to the [Android Epic](https://mozilla-hub.atlassian.net/browse/FXDROID-1965) but this is open for anyone to do)

Reply:
I find some benefit in it. I learned a lot about mobile messaging from testing around the survey example, so I find it super helpful for those who are unfamiliar with the area. But open to hear otherwise.

Ah yes thank you! That documentation is missing a lot on iOS side (i.e. list of triggers). I'll create a ticket for it too. Thanks for bringing it up!

Reply:
Perhaps in the future we could include an 'examples' supporting doc for this file to prevent the manifest file becoming too large - or a link to the experimenter docs to ensure we are staying in sync - it's easy for these to become out of sync if folks only start updating the 'code' version

Reply:
sounds good to me, this file is small now, but happy to create a ticket and work together on this if you would like for the future! A link in the docs makes sense to me for now.